bin/custom/concerttours/web/webroot/WEB-INF/views/bandDetails.jsp

```diff
@@ -5,6 +5,9 @@
 <body>
 <h1>Band Details</h1>
```

Comment:
please, put all string values in jsp's under localization properties and load on frontend with spring:message